docs/pbc/all/data-exchange/202404.0/spryker-middleware-powered-by-alumio/incrementally-import-products-with-spryker-middleware-powered-by-alumio.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Incrementally import recently updated products with Spryker Middleware Powered by Alumio
+description: Learn how to incrementally import products that have been updated a day ago
+last_updated: Apr 18, 2024
+template: howto-guide-template
+---
+
+This document describes how to import products that have recently been updated. Following the described procedure, you can import products that have been updated within the past day.
+
+To run the incremental import of products, follow these steps.
+
+## 1. Configure the Spryker Middleware Powered by Alumio connection with Akeneo PIM and SCCOS
+
+* If you create the new configuration, follow [Configure the Spryker Middleware Powered by Alumio connection with Akeneo PIM and SCCOS](/docs/pbc/all/data-exchange/{{page.version}}/spryker-middleware-powered-by-alumio/integration-apps/akeneo-pim-integration-app/configure-the-akeneo-pim-integration-app/configure-the-smpa-connection-with-akeneo-pim-and-sccos.html).
+* If you update the existing configuration, proceed with [3. Create an incoming configuration](#3-create-an-incoming-configuration).
+
+## 2. Configure the data mapping between Akeneo and SCCOS
+
+* If you create the new configuration, follow the procedure described in [If you create the new configuration, follow the procedure described in](/docs/pbc/all/data-exchange/{{page.version}}/spryker-middleware-powered-by-alumio/integration-apps/akeneo-pim-integration-app/configure-the-akeneo-pim-integration-app/configure-data-mapping-between-akeneo-and-sccos.html).
+* If you update the existing configuration, proceed with [3. Create an incoming configuration](#3-create-an-incoming-configuration).
+
+## 3. Create an incoming configuration
+
+1. In Spryker Middleware powered by Alumio, go to **Connections** > **Incoming** and click **+**.
+  This opens the **Create an incoming configuration** page.
+2. For **Name**, enter the name for this configuration.
+  As you are entering the name, the **Identifier** is automatically populated.
+3. Optional: For **Description**, add the description of the configuration.
+4. To activate the incoming configuration after creating, make sure **Status** is set to **Enabled**.
+5. For **Subscriber**, enter and select **HTTP Subscriber**.
+6. For **Request URI**, enter `/api/rest/v1/products`.
+7. For **Payload type**, enter and select **Encoded data**.
+8. For **Request Parameters**, select **string** and enter the following string: `search={"updated":[{"operator":">","value":"&{lastTimestamp}"}]}&limit=10`. Make sure to define the `limit` value with the number of products you want to import. If you want to import all updated products, see [Batch products import](/docs/pbc/all/data-exchange/{{page.version}}/spryker-middleware-powered-by-alumio/integration-apps/akeneo-pim-integration-app/configure-the-akeneo-pim-integration-app/configure-the-data-integration-path-between-akeneo-and-sccos.html#batch-products-import).
```

Comment:
We need to remove steps7 and 8 of  3. Create an incoming configuration It was discussed in https://spryker.atlassian.net/browse/DEX-332